base/applications/rapps/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ add_rc_deps(rapps.rc ${rapps_rc_deps})
 add_executable(rapps ${SOURCE} rapps.rc)
 set_module_type(rapps win32gui UNICODE)
 target_link_libraries(rapps uuid wine)
-add_importlibs(rapps advapi32 comctl32 gdi32 wininet user32 shell32 shlwapi ole32 setupapi msvcrt kernel32 ntdll)
+add_importlibs(rapps advapi32 comctl32 gdi32 wininet user32 shell32 shlwapi ole32 setupapi msvcrt kernel32 ntdll gdiplus)
```

Comment:
OK, I've changed it. but... may I ask why? 🤣

Reply:
Thomas explained this nicely in mattermost:

> The order is important because if ntdll comes first it'll satisfy all the CRT functions, so you're linking against the NT CRT instead of the Win32 one
And because msvcrt includes msvcrtex, which may need stuff from kernel32 and ntdll
Having them last... Hmm. It actually breaks something (at link time) if you don't, but I don't remember what
(but not in all modules)
But the underlying concept is: linking has an order, and you add new things on the left, not the right

Reply:
now I see ... thanks!!!